File: latihanB/python/Processor.py
class Processor:
    def __init__(self, name="", price=0):
        self.__name = name
        self.__price = price

    def setName(self, name=""):
        self.__name = name

    def getName(self):
        return self.__name

    def setPrice(self, price=0):
        self.__price = price

    def getPrice(self):
        return self.__price

    def printProcessor(self):
        print("Name\t\t: " + self.__name)
        print("Price\t\t: " + str(self.__price) + " IDR")

    

File: latihanB/java/Processor.java
public class Processor {
    private String name;
    private int price;

    Processor() { }

    Processor(String name, int price) {
        this.name = name;
        this.price = price;
    }

    public void setName(String name) {
        this.name = name;
    }

    public String getName() {
        return this.name;
    }

    public void setPrice(int price) {
        this.price = price;
    }

    public int getPrice() {
        return this.price;
    }

    public void printProcessor() {
        System.out.println("Name\t\t: " + this.name);
        System.out.println("Price\t\t: " + this.price + " IDR");
    }
}